src/ensembl/src/shared/toggle-image-button/ImageHolder.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { ImageButtonStatus } from 'src/shared/toggle-image-button/ToggleImageButton';
+
+type Props = {
+  buttonStatus: ImageButtonStatus;
+  image: any;
+  imageStyles: any;
```

Comment:
This should also be possible to type more precisely:

```
classNames: { [key: keyof ImageButtonStatus ]: string }
```
?

Reply:
Not sure if this still applies as we will be passing multiple classNames?